### ISSUE TITLE
[BUG]IO error when simultaneous playback -capture on ADLP_RVP_NOCODEC/TGLU_RVP_NOCODEC_MULTICORE

### ISSUE THREAD

**Issue by XiaoyunWu6666** [thesofproject/sof#4458]
**Describe the bug**
IO error happened when simultaneous playback capture and multiple-pause-resume on ADLP_RVP_NOCODEC in daily-ADL 5117
Even with DSP panic when TRIG_START in simultaneous playback and capture .
However , when I tried the case manually , this usually led the DUT to reboot and lost connection.

**To Reproduce**
TPLG=sof-tgl-nocodec-ci.tplg ~/sof-test/test-case/simultaneous-playback-capture.sh -l 50

**Environment**
Kernel Branch: topic/sof-dev
Kernel Commit: 6bd3089a
SOF Branch: main
SOF Commit: a7604433c2a2

**Screenshots or console output**
[dmesg]
```shell
[ 5204.620589] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx: 0x60040000: GLB_STREAM_MSG: TRIG_START
[ 5204.620943] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: ipc tx succeeded: 0x60040000: GLB_STREAM_MSG: TRIG_START
[ 5204.627920] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error : DSP panic!
[ 5204.627925] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: panic: dsp_oops_offset 788480 offset 788480
[ 5204.627931] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: status: fw entered - code 00000005
[ 5204.628026] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: runtime exception
[ 5204.628028] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: trace point 00004000
[ 5204.628030] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: panic at :0
[ 5204.628033] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: DSP Firmware Oops
[ 5204.628035] kernel: sof-audio-pci-intel-tgl 0000:00:1f.3: error: Exception Cause: LoadProhibitedCause, A load referenced a page mapped with an attribute that does not permit loads
```

**Comment by XiaoyunWu6666** [comment]
does it have something to do with #4328

**Comment by mengdonglin** [comment]
This issue isn't reproduced in daily test report 5215. I guess it was fixed by PR #4462.

**Comment by XiaoyunWu6666** [comment]
> This issue isn't reproduced in daily test report 5215 and 5207. I guess it was fixed by PR #4462. @XiaoyunWu6666 Can you help double check and see if we can close this issue?

haven't happened in daily tests for quite long

**Comment by lgirdwood** [comment]
Lets monitor and close if not reproducing over next few days.

**Comment by keqiaozhang** [comment]
Confirmed that this issue is fixed with latest main and stable v2.0 branch.
